crates/rover-client/src/blocking/client.rs

```diff
@@ -0,0 +1,65 @@
+use crate::headers;
+use crate::RoverClientError;
+use graphql_client::GraphQLQuery;
+
+/// Represents a client for making http requests.
+pub struct Client {
+    api_key: String,
+    client: reqwest::blocking::Client,
+    uri: String,
+}
+
+impl Client {
+    /// Construct a new [Client] from 2 strings, an `api_key` and an optional `uri`.
+    /// if the uri is None, we'll use the default uri for apollo studio
+    pub fn new(api_key: String, uri: String) -> Client {
```

Comment:
The comment was old. I was originally doing the default logic in the `new` function, but I instead just made the `uri` required and removed the default uri logic for now. I probably want to move the default logic to the session where we initialize everything rather than here, just so if there's any other use case for this `Client`, it wouldn't have the same defaults.